src/subdomains/generic/user/models/account-merge/account-merge.service.ts

```diff
@@ -82,9 +82,16 @@ export class AccountMergeService {
     if (request.isExpired) throw new BadRequestException('Merge request is expired');
     if (request.isCompleted) throw new ConflictException('Merge request is already completed');
 
-    await this.userDataService.mergeUserData(request.master.id, request.slave.id);
+    const [master, slave] =
+      request.master.kycLevel < request.slave.kycLevel
+        ? [request.slave, request.master]
+        : [request.master, request.slave];
```

Comment:
Could actually do this 😁 
`const [master, slave] = [request.master, request.slave].sort((a, b) => b.kycLevel - a.kycLevel);`